CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * Fixed a crash when calling `Table.toString()` in debugger (#2429).
 * Fixed a race condition which would cause some `RealmResults` to not be properly updated inside a `RealmChangeListener`. This could result in crashes when accessing items from those results (#2926/#2951).
 * Fixed a bug that could cause Realm to lose track of primary key when using `RealmObjectSchema.removeField()` and `RealmObjectSchema.renameField()` (#2829).lts (#2926).
+* Fixed a bug that some async related JNI functions cannot be found on some devices due to wrong declaration.
```

Comment:
Fixed a bug that prevented some devices from finding async related JNI methods correctly.